Apps/W1/EDocument/app/src/InvoicePreview/EInvoiceLine.Table.al

```diff
@@ -0,0 +1,53 @@
+table 6100 "E-Invoice Line"
+{
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "E-Document Entry No."; Integer)
+        {
+            TableRelation = "E-Document";
+            Caption = 'E-Document Entry No';
+        }
+        field(2; "Line No."; Integer)
+        {
+            Caption = 'Line No.';
+        }
+        field(3; "No."; Code[20])
+        {
+            Caption = 'No.';
+        }
+        field(4; Description; Text[100])
+        {
+            Caption = 'Description';
+        }
+        field(5; "Unit of Measure Code"; Text[50])
```

Comment:
This field is assigned with the field `field(5407; "Unit of Measure Code"; Code[10])` therefore I think that it should be `Code[10 ]`